.travis/apisix_cli_test.sh

```diff
@@ -211,8 +211,40 @@ fi
 
 echo "passed: worker_shutdown_timeout in nginx.conf is ok"
 
+# empty allow_admin in conf/config.yaml
```

Comment:
I think the title is not a good title. Because the following example has a situation where ʻallow_admin` is not empty.

Reply:
nice catch